<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteOutlier</class>
 <widget class="QFrame" name="deleteOutlier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="SimpleCardWidget" name="deleteOutlierWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="HorizontalSeparator" name="HorizontalSeparator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>361</width>
      <height>3</height>
     </rect>
    </property>
   </widget>
   <widget class="StrongBodyLabel" name="saveNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>曲线保存名称</string>
    </property>
   </widget>
   <widget class="LineEdit" name="saveNameLineEdit">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>240</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>初始点坐标</string>
    </property>
    <widget class="BodyLabel" name="xLabel_L">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="LineEdit" name="xLineEdit_L">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="BodyLabel" name="yLabel_L">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="LineEdit" name="yLineEdit_L">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>101</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="CaptionLabel" name="ymistake_L">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>必填项</string>
     </property>
    </widget>
    <widget class="CaptionLabel" name="xmistake_L">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>必填项</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_R">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>结束点坐标</string>
    </property>
    <widget class="BodyLabel" name="xLabel_R">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="LineEdit" name="xLineEdit_R">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="BodyLabel" name="yLabel_R">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="LineEdit" name="yLineEdit_R">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>101</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="CaptionLabel" name="ymistake_R">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>必填项</string>
     </property>
    </widget>
    <widget class="CaptionLabel" name="xmistake_R">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>必填项</string>
     </property>
    </widget>
   </widget>
   <widget class="PrimaryPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="PrimaryPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="ProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>361</width>
      <height>4</height>
     </rect>
    </property>
   </widget>
   <widget class="CaptionLabel" name="mistake3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>必填项</string>
    </property>
   </widget>
   <widget class="CaptionLabel" name="mistakeOkButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>必填项</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
